docs-linting/changelog.md

```diff
@@ -1,9 +1,14 @@
 # Change Log
 
-## 0.0.2 (2/25/2020, 2018)
+## 0.0.4 (06/04/2020)
```

Comment:
Hi @bharney0 , do you think we should change the dates format to match the other extensions i.e. June 4th, 2020?